target_chains/ethereum/contracts/contracts/pyth/PythGovernanceInstructions.sol

```diff
@@ -82,6 +83,12 @@ contract PythGovernanceInstructions {
         uint newFee;
     }
 
+    struct WithdrawFeePayload {
+        address targetAddress;
+        // Amount in wei, matching the native uint256 type used for address.balance in EVM
+        uint256 amount;
+    }
```

Comment:
in hindsight i should say what you have is actually nicer, but i want to be compatible with the previous methods. Can we change it to have base and exponent like other fee instructions?